runtime/lib/component/app-loader.js

```diff
@@ -30,6 +30,7 @@ function AppChargeur (runtime) {
   this.appManifests = {}
 
   this.notifications = {
+    'on-boot': [],
```

Comment:
Could you describe:
1. What's the purpose of `on-boot` ? 
2. What's the opportunity the notification occurs?

Reply:
Description of the notification channel is mentioned in the PR.

Reply:
I guess @morrowind means documentation for these notifications, this PR seems urgent to be done, let's revisit that docs later :)

Reply:
> Description of the notification channel is mentioned in the PR.

`boot` is a duration not a moment conception,  that's why I ask question (2). Because `boot` may lasts a long time.
And for question (1), I mean what case what app need to listen `on-boot` event?

Reply:
For Q1, since currently long-timed timers/alarms are set in apps rather than delegated in VUI, apps have to re-set timers after boot to correctly work.

Reply:
> For Q1, since currently long-timed timers/alarms are set in apps rather than delegated in VUI, apps have to re-set timers after boot to correctly work.

Thus, I think I may partially understand the purpose.
The `on-boot` event will occur after boot completed regardless of network connection status. But the `on-ready` event will only occur in condition of network has been connected. Is it right?
If so, I suggest the event name can be changed to `on-boot-completed`/`on-system-ready`. If not so, please give more explanation.

Reply:
Yes, you got it right. /cc @yorkie do you have any naming suggestion?

Reply:
I'm +1 on `on-boot-completed`.

Reply:
since word boot is a verb, I've changed the name to `on-system-booted`.